app/controllers/api/v1/reports_controller.rb

```diff
@@ -38,7 +37,7 @@ def destroy
       def last
         conditions = { :host_id => Host.authorized(:view_hosts).friendly.find(params[:host_id]).try(:id) } if params[:host_id].present?
         max_id = resource_scope.where(conditions).maximum(:id)
-        @report = resource_scope.includes(:logs => [:message, :source]).find(max_id)
+        @report = resource_scope.find(max_id)
```

Comment:
Given the issues I've seen, I would like to keep the pre-loading out, even for a single report, I'm not sure `N + 1` matter much, when `N = 1`.